src/test/java/uk/gov/companieshouse/search/api/service/search/AlphabeticalSearchIndexServiceTest.java

```diff
@@ -138,7 +139,7 @@ public void testSuccessfulSearchFound() throws IOException {
 
         assertEquals(SEARCH_FOUND, responseObject.getStatus());
         assertEquals(TOP_HIT, searchResults.getTopHit());
-        assertEquals(20, searchResults.getResults().size());
+        assertEquals(19, searchResults.getResults().size());
     }
```

Comment:
Why has the size of the search results changed?

Reply:
I have removed this assertion as it will show different outcome if run against a different database or if there is some data change.